api/circulation.py

```diff
@@ -32,7 +32,16 @@ def fd(self, d):
 
 class FulfillmentInfo(CirculationInfo):
 
-    """A record of an attempt to fulfil a loan."""
+    """A record of an attempt to fulfill a loan.
+
+    :param identifier_type Ex: Third party provider, ISBN, URI, etc...
+    :param identifier Contains ISBN or third party item id, etc., and links to LicensePool, Work, etc..
+    :param content_link Either URL to download ACSM file from or URL to streaming content.
+    :param content_type Media type of the book version we're getting.  
+        Ex: "text/html" or Representation.TEXT_HTML_MEDIA_TYPE + DeliveryMechanism.STREAMING_PROFILE .
```

Comment:
This could be any media type. If you're giving examples, another example would be EPUB_MEDIA_TYPE.